bundles/com.salesforce.bazel.eclipse.core/plugin.xml

```diff
@@ -4,14 +4,14 @@
    <extension
          point="org.eclipse.jdt.core.classpathContainerInitializer">
       <classpathContainerInitializer
-            class="com.salesforce.bazel.eclipse.classpath.BazelClasspathContainerInitializer"
+            class="com.salesforce.bazel.eclipse.classpath.BazelCoreClasspathContainerInitializer"
```

Comment:
By making the classpath initializer live in the Core plugin, we ensure the Core plugin is activated before we try to initialize the classpath containers.